backend/app/models/sys_menu.py

```diff
@@ -22,6 +22,7 @@ class Menu(Base):
     icon: Mapped[str | None] = mapped_column(String(100), default=None, comment='菜单图标')
     path: Mapped[str | None] = mapped_column(String(200), default=None, comment='路由地址')
     menu_type: Mapped[int] = mapped_column(default=0, comment='菜单类型（0目录 1菜单 2按钮）')
+    meta: Mapped[dict | None] = mapped_column(JSON, default=None, comment='菜单元数据')
```

Comment:
This may seem strange, as the meta contains a lot of information that becomes uncontrollable when debugging, and it is a design that has not been used in regular menus.

I think we should modify the frontend code to do the adaptation